scripts/commands/updateApiDocs.ts

```diff
@@ -128,10 +128,9 @@ async function prepareSphinxFolder(pkg: Pkg, args: Arguments): Promise<string> {
 
 async function deleteExistingMarkdown(pkg: Pkg): Promise<void> {
   const markdownDir = pkg.outputDir("docs");
-  if (pkg.isLatest() || (await pathExists(markdownDir))) {
```

Comment:
I don't think it's necessary to check if `pkg.isLatest()` - it's already covered by the `pathExists` check. If the path does not exist, then there is nothing to delete, regardless of latest or not.